cloud/aws/deploy/batch.go

```diff
@@ -249,7 +249,6 @@ func (p *NitricAwsPulumiProvider) Batch(ctx *pulumi.Context, parent pulumi.Resou
 	}
 
 	listActions := []string{
-		// TODO: test that all resources still work without these permissions
```

Comment:
we might be able to remove some of these permissions since re-working to use ssm